src/main/java/com/laserfiche/api/client/apiserver/TokenClient.java

```diff
@@ -3,19 +3,27 @@
 import com.laserfiche.api.client.model.CreateConnectionRequest;
 import com.laserfiche.api.client.model.SessionKeyInfo;
 
+/**
+ * The Laserfiche Self-Hosted token route API client.
+ */
 public interface TokenClient extends AutoCloseable {
 
     /**
-     * @param repoId Repository name
```

Comment:
Maybe I'm missing something, but isn't it better to document it as "Repository ID"? cause the first thing that comes to a client's mind is why we haven't named the parameter repoName?

Reply:
changed it to Repository ID